test/unit/driver-specs.js

```diff
@@ -18,12 +19,36 @@ describe('driver commands', () => {
     proxySpy.reset();
   });
 
-  describe('status', () => {
-    it('should send status request to WDA', async () => {
-      await driver.getStatus();
-      proxySpy.calledOnce.should.be.true;
-      proxySpy.firstCall.args[0].should.eql('/status');
-      proxySpy.firstCall.args[1].should.eql('GET');
+  describe('status caching', () => {
```

Comment:
This test fails (https://travis-ci.org/appium/appium-xcuitest-driver/jobs/204494918#L2438). Do you not run the pre-commit hooks before committing? This should have been caught there.